Makie/src/compute-plots.jl

```diff
@@ -682,7 +678,6 @@ function connect_plot!(parent::SceneLike, plot::Plot{Func}) where {Func}
         end
     end
 
-    # TODO, do this for recipes?
```

Comment:
This function (`connect_plot!()`) runs for recipes now, right? So this is outdated?